### PR TITLE
USWDS 3.0 - "Introducing USWDS 3.0" edits

### PR DESCRIPTION
**Pull request by amyleadem**
## Preview link
[Introducing USWDS 3.0 post](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.app.cloud.gov/preview/uswds/uswds-site/al-intro-edits/whats-new/updates/2022/04/28/introducing-uswds-3-0/)

## Description

Edits for the "Introducing USWDS 3.0" post

### CODE REVIEW

**Comment by amyleadem** [comment]
@thisisdano 
I am not able to find the content for the link to documentation/developers/#long-term-support-of-v2x in [line 71](https://github.com/uswds/uswds-site/blob/5c62a2f411273df45fbdc1cbf082098d42086359/_posts/2022-04-28-introducing-uswds-3-0.md?plain=1#L71). I see only support for v1.x on that page.

File: _posts/2022-04-28-introducing-uswds-3-0.md [hunk 1]
```diff
@@ -58,14 +58,14 @@ IE11 has fallen below our 2% support threshold as monitored by analytics.usa.gov
...
 
 We believe that JavaScript-disabled environments fall below our 2% support threshold, and that we may require JavaScript for some design system functionality. While we have not made any changes to our codebase that affect how we use JavaScript with our components, we may make changes that require JavaScript for some functionality as we move forward. When we do make any changes like this, we'll be sure to note it in our release notes.
 
 ### Support USWDS 2 with maintenance and security patches for one year
 
-In addition to supporting active development on USWDS 3, We will continue to support the USWDS 2 branch for one year of maintenance and security patches (until April 2023). The current LTS version of USWDS 2 is [USWDS 2.13.3](https://github.com/uswds/uswds/discussions/4626).
+In addition to supporting active development on USWDS 3, we will continue to support the USWDS 2 branch for one year of maintenance and security patches (until April 2023). The current LTS version of USWDS 2 is [USWDS 2.13.3](https://github.com/uswds/uswds/discussions/4626).
 
 [Long-term support of version 2.x]({{ site.baseurl }}/documentation/developers/#long-term-support-of-v2x) →
```
Comment (amyleadem) [inline _posts/2022-04-28-introducing-uswds-3-0.md:71]: @thisisdano
I am not able to find the content for this link

File: _posts/2022-04-28-introducing-uswds-3-0.md [hunk 2]
```diff
@@ -19,24 +19,24 @@ For most USWDS 2 projects, migrating to USWDS 3.0 should only require changing
...
-[Read more about the Sass Module syntax](https://sass-lang.com/) → 
+[Read more about the Sass module syntax](https://sass-lang.com/)
 
 ### Improve performance and reduce the size of your project CSS 
 
-Unbundle the design system with USWDS 3.0. Using USWDS 3.0 with Sass Module syntax allows teams to unbundle their implementations of USWDS and use only the components and code that they need on their project. Depending on your project, this could mean significant reduction of CSS code you ship. This means faster load times, better scores in performance evaluation tools, and a better developer experience. 
+Unbundle the design system with USWDS 3.0. Using USWDS 3.0 with Sass module syntax allows teams to unbundle their implementations of USWDS and use only the components and code that they need on their project. Depending on your project, this could mean significant reduction of CSS code you ship. This means faster load times, better scores in performance evaluation tools, and a better developer experience. 
 
-[Read more about USWDS Packages](#) → 
+[Read more about USWDS Packages](/components/packages/) →
```
Comment (thisisdano) [inline _posts/2022-04-28-introducing-uswds-3-0.md:39]:
```suggestion
[Read more about USWDS Packages]({{ site.baseurl }}/components/packages/) →
```

File: _posts/2022-04-28-introducing-uswds-3-0.md [hunk 3]
```diff
@@ -58,14 +58,14 @@ IE11 has fallen below our 2% support threshold as monitored by analytics.usa.gov
 
 [USWDS browser support (2% rule)](https://designsystem.digital.gov/documentation/developers/#browser-support) →
```
Comment (thisisdano) [inline _posts/2022-04-28-introducing-uswds-3-0.md:59]:
```suggestion
[USWDS browser support (2% rule)]({{ site.baseurl }}/documentation/developers/#browser-support) →
```

File: _posts/2022-04-28-introducing-uswds-3-0.md [hunk 4]
```diff
@@ -58,14 +58,14 @@ IE11 has fallen below our 2% support threshold as monitored by analytics.usa.gov
...
 
 ### Support some Javascript-required functionality 
 
 We believe that JavaScript-disabled environments fall below our 2% support threshold, and that we may require JavaScript for some design system functionality. While we have not made any changes to our codebase that affect how we use JavaScript with our components, we may make changes that require JavaScript for some functionality as we move forward. When we do make any changes like this, we'll be sure to note it in our release notes.
 
 ### Support USWDS 2 with maintenance and security patches for one year
 
-In addition to supporting active development on USWDS 3, We will continue to support the USWDS 2 branch for one year of maintenance and security patches (until April 2023). The current LTS version of USWDS 2 is [USWDS 2.13.3](https://github.com/uswds/uswds/discussions/4626).
+In addition to supporting active development on USWDS 3, we will continue to support the USWDS 2 branch for one year of maintenance and security patches (until April 2023). The current LTS version of USWDS 2 is [USWDS 2.13.3](https://github.com/uswds/uswds/discussions/4626).
```
Comment (thisisdano) [inline _posts/2022-04-28-introducing-uswds-3-0.md:69]:
```suggestion
In addition to supporting active development on USWDS 3, we will continue to support the USWDS 2 branch for one year of maintenance and security patches (until April 2023). The current LTS version of USWDS 2 is [USWDS 2.13.3](https://github.com/uswds/uswds/releases/tag/v2.13.3).
```

Let's link to the release, not the discussion

File: _posts/2022-04-28-introducing-uswds-3-0.md (hunk 1 above)
Comment (thisisdano) [reply _posts/2022-04-28-introducing-uswds-3-0.md:71]: It should be there — possibly related to caching of the old README?

Screen Shot 2022-04-27 at 3 38 39 PM

**Review by thisisdano — CHANGES_REQUESTED** [review]
One more change: This first paragraph should read `USWDS 3.0 is an important major...` (Add `3.0`)

File: _posts/2022-04-28-introducing-uswds-3-0.md (hunk 4 above)
Comment (amyleadem) [reply _posts/2022-04-28-introducing-uswds-3-0.md]: Updated in [769f8c3](https://github.com/uswds/uswds-site/pull/1561/commits/769f8c3ac529340c57d075dcaf71a70af01e120c)

File: _posts/2022-04-28-introducing-uswds-3-0.md (hunk 2 above)
Comment (amyleadem) [reply _posts/2022-04-28-introducing-uswds-3-0.md]: Updated in [769f8c3](https://github.com/uswds/uswds-site/pull/1561/commits/769f8c3ac529340c57d075dcaf71a70af01e120c)

File: _posts/2022-04-28-introducing-uswds-3-0.md (hunk 3 above)
Comment (amyleadem) [reply _posts/2022-04-28-introducing-uswds-3-0.md]: Updated in [769f8c3](https://github.com/uswds/uswds-site/pull/1561/commits/769f8c3ac529340c57d075dcaf71a70af01e120c)

File: _posts/2022-04-28-introducing-uswds-3-0.md (hunk 1 above)
Comment (amyleadem) [reply _posts/2022-04-28-introducing-uswds-3-0.md:71]: My bad. I cleared the ole jekyll cache and all was well. Marking this as resolved.

**Review by thisisdano — APPROVED** [review]

**Pull request merged**